src/NLU.DevOps.Luis/LuisClient.cs

```diff
@@ -111,9 +112,10 @@ public Task DeleteAppAsync(string appId, CancellationToken cancellationToken)
             return this.AuthoringClient.Apps.DeleteAsync(Guid.Parse(appId), cancellationToken);
         }
 
-        public Task<IList<ModelTrainingInfo>> GetTrainingStatusAsync(string appId, string versionId, CancellationToken cancellationToken)
+        public async Task<IEnumerable<string>> GetTrainingStatusAsync(string appId, string versionId, CancellationToken cancellationToken)
```

Comment:
Thoughts on creating a singular enum here instead, and removing the IEnumerable? Ultimately we only care whether or not the entire training operation is a) Queued/InProgress, b) Failed, or c) Successful.